<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginRegister</class>
 <widget class="QMainWindow" name="LoginRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>100</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LoginRegister</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 250, 250);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QTabBar::Tab {
	width: 150;
	height: 40;
	border: 1px solid black;
	border-radius: 4

}
QTabWidget::Pane {
	background-color: #fafafa;
}
QTabBar::Tab::Selected{
	background-color: #d9c57e;


}
QLineEdit {
	background-color: none;
	border: 1px solid black;
 	border-radius: 4;

}

QPushButton {
	border: 1px solid black;
	background-color: #d9c57e;
	border-radius: 4;


}


</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>280</y>
      <width>304</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="Login">
     <attribute name="title">
      <string>Login</string>
     </attribute>
     <widget class="QLineEdit" name="loginUsername">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>34</y>
        <width>304</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 300</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="loginPassword">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>304</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 300</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>103</x>
        <y>149</y>
        <width>100</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Register</string>
     </attribute>
     <widget class="QLineEdit" name="registerEmail">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>34</y>
        <width>304</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 300</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerUsername">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>304</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 300</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerPassword">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>304</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 300</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="registerPasswordConfirm">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>304</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 300</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Confirm Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="registerButton">
      <property name="geometry">
       <rect>
        <x>102</x>
        <y>270</y>
        <width>100</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/img/SmallCardVaultLogo.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
